node/index.js

```diff
@@ -582,6 +582,12 @@ TChannelConnection.prototype.onReaderError = function onReaderError(err) {
         localName: self.channel.hostPort,
         error: err
     });
+
+    // TODO instead of resetting send an error frame first.
```

Comment:
I think this is a TODO for resetAll in general: when resetting with an error, send an appropriate error frame first?